scripts/create_tables.sql

```diff
@@ -1 +1,30 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY
+  start_on DATE,
```

Comment:
This column and the next column should be named `date` and `time`, respectively.  There won't be any issues since we're only using them as column names and not identifier names.